exercises/mongo-shell/intermediate-read.md

```diff
@@ -108,6 +119,58 @@ Use the `$nin` query operator (**not in** operator) to match **none** of the val
 
 ---
 
+## Logical Operators
+
+### The "$or" operator
+This operator **OR**s the filters stated in a query and returns documents where atleast 1 of the filters is true. 
+
+**Exercise 1** :computer: 
+
+Write a query to list movies whose `imdb.rating` is greater than 8.5 or `metacritic` rating is greater than 85.
+
+:arrow_right: Note that the `$or` operator takes an **array** of values. 
+
+**query:**
+```javascript
+db.movieDetails.find({"$or": [
+    {"imdb.rating": {"$gt": 8.5}}, 
+    {"metacritic": {"$gt": 85}}]}, 
+        {"_id": 0, "title": 1, "imdb.rating": 1, "metacritic": 1})
+```
+---
+
+### The "$and" operator
+
+As you already know, MongoDB implicitly **AND**s the filters when separated by `,` in the query. 
```

Comment:
add a link here, refer to the location in the document that explained this earlier, just for quick navigation.  Cross-links are usually always a good idea.

```suggestion
As you [already know](../blob/master/exercises/mongo-shell/basic-crud/basic-read.md#Filtering-on-multiple-fields), MongoDB implicitly **AND**s the filters when separated by `,` in the query. 
```

This makes use of a relative link with github, similar to you navigating your filesystem which is where the `../blob/master/` comes from.  Also **any** heading (h1, h2, h3 and so on) can be used as a link target.  

In this case, your target heading has spaces in it.  When that is the case you need to include `-` in place of the space inside of the link URL.  You can see how I did that above.